.ng-dev/config.ts

```diff
@@ -0,0 +1,17 @@
+import {GithubConfig} from '../ng-dev/utils/config';
+import {FormatConfig} from '../ng-dev/format/config';
+
+const format: FormatConfig = {
+  'prettier': true,
+  'buildifier': true,
+};
+
+const github: GithubConfig = {
+  owner: 'angular',
+  name: 'dev-infra',
+};
+
+module.exports = {
+  format,
+  github,
+};
```

Comment:
I see that you have `format` and `github` set here, but also have individual files. I think we should just re-export the exports from `format.ts` and `github.ts` right?

Reply:
Whoops. Fixed.